backend/monitoring/dashboard.py

```
@@ -707,6 +732,343 @@ async def trigger_test(test_type: str, admin=Depends(get_current_admin_dependenc
+
+@router.post("/business-logic-validate")
+async def trigger_business_logic_validation(request: dict, admin=Depends(get_current_admin_dependency)):
+    """Trigger business logic validation for spiritual content"""
+    try:
+        from monitoring.business_validator import BusinessLogicValidator
+        
+        validator = BusinessLogicValidator()
+        
+        # Get validation request parameters
+        session_context = request.get("session_context", {})
+        validation_type = request.get("validation_type", "full")
```

Ruff (0.12.2)

Local variable `validation_type` is assigned to but never used

Remove assignment to unused variable `validation_type`

(F841)